Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    camt_parser (2.11.0)
+    camt_parser (2.11.1)
```

Comment:
This was the diff which stopped the `bundle install` in CI. Now, we'll have no issues.

Reply:
Ah, damn, I broke that the other day then 😄 
Thank you!